recipes/norwegianblue/recipe.yaml

```diff
@@ -0,0 +1,67 @@
+schema_version: 1
+
+context:
+  name: norwegianblue
+  version: "0.22.0"
+  python_min: "3.9"
```

Comment:
Setting this explicitly even though this is the current `python_min` as the library won't necessarily move forward with dropping Python lower bounds at the same rate as conda-forge does.